### PR TITLE
Adding test case OCS-347

### PR DESCRIPTION
**Pull request by parikshithb**
Made changes to my test script with latest changes done in OCS-CI. 
Log: http://pastebin.test.redhat.com/770756

### CODE REVIEW

File: tests/manage/test_rbd_csi_default_sc.py [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+    setup(self)
+
+
+def setup(self):
+    """
+    Setting up the environment for the test
+    """
+    global SECRET
+    self.rbd_secret = defaults.CSI_RBD_SECRET.copy()
+    del self.rbd_secret['data']['kubernetes']
```
Comment (ebenahar) [inline tests/manage/test_rbd_csi_default_sc.py]: use helpers.create_secret()

File: tests/manage/test_rbd_csi_default_sc.py [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+    log.info("Deleting PVC")
+    PVC.delete()
+    log.info("Deleting StorageClass")
+    STORAGE_CLASS.delete()
+    log.info("Deleting Secret")
+    SECRET.delete()
+
+
+def get_admin_key_from_ceph_tools():
+    """
```
Comment (ebenahar) [inline tests/manage/test_rbd_csi_default_sc.py]: this is already implemented in helpers.get_admin_key()

File: tests/manage/test_rbd_csi_default_sc.py [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+    )
+
+    def test_ocs_347(self):
+        """
+        Testing default storage class creation and pvc creation using rbd
+        """
+        global PVC, STORAGE_CLASS
+        rbd_sc = defaults.CSI_RBD_STORAGECLASS_DICT.copy()
+        rbd_sc['parameters']['monitors'] = self.mons
+        STORAGE_CLASS = OCS(**rbd_sc)
```
Comment (ebenahar) [inline tests/manage/test_rbd_csi_default_sc.py]: use helpers.create_storage_class()

File: tests/manage/test_rbd_csi_default_sc.py (hunk 1 above)
Comment (parikshithb) [reply tests/manage/test_rbd_csi_default_sc.py]: Done

File: tests/manage/test_rbd_csi_default_sc.py (hunk 2 above)
Comment (parikshithb) [reply tests/manage/test_rbd_csi_default_sc.py]: Done

File: tests/manage/test_rbd_csi_default_sc.py (hunk 3 above)
Comment (parikshithb) [reply tests/manage/test_rbd_csi_default_sc.py]: Done

**Comment by parikshithb** [comment]
=============================================================================================== test session starts ===============================================================================================
platform linux -- Python 3.7.3, pytest-4.6.2, py-1.8.0, pluggy-0.12.0 -- /home/pbyregow/venv/bin/python3
cachedir: .pytest_cache
rootdir: /home/pbyregow/git/pytest/ocs-ci, inifile: pytest.ini
plugins: reportportal-1.0.5
collected 1 item                                                                                                                                                                                                  

tests/manage/test_rbd_csi_default_sc.py::TestCaseOCS347::test_ocs_347 
------------------------------------------------------------------------------------------------- live log setup --------------------------------------------------------------------------------------------------
14:21:26 - MainThread - root - INFO - All logs are located under /tmp/logs
14:21:26 - MainThread - tests.manage.test_rbd_csi_default_sc - INFO - Creating RBD Secret
14:21:26 - MainThread - utility.utils - INFO - Executing command: oc -n openshift-storage --kubeconfig /home/pbyregow/Music/auth/kubeconfig get Pod --selector=app=rook-ceph-tools -o yaml
14:21:28 - MainThread - utility.utils - INFO - Executing command: oc -n openshift-storage --kubeconfig /home/pbyregow/Music/auth/kubeconfig rsh rook-ceph-tools-76c7d559b6-r6m89 ceph auth get-key client.admin --format json-pretty
14:21:33 - MainThread - resources.ocs - INFO - Adding Secret with name secret-test-1314213323
14:21:33 - MainThread - utility.utils - INFO - Executing command: oc -n openshift-storage --kubeconfig /home/pbyregow/Music/auth/kubeconfig create -f /tmp/Secret7x9k6cv1 -o yaml
14:21:34 - MainThread - utility.utils - INFO - Executing command: oc -n openshift-storage --kubeconfig /home/pbyregow/Music/auth/kubeconfig get Secret secret-test-1314213323 -o yaml
-------------------------------------------------------------------------------------------------- live log call --------------------------------------------------------------------------------------------------
14:21:36 - MainThread - tests.manage.test_rbd_csi_default_sc - INFO - Creating RBD StorageClass
14:21:36 - MainThread - resources.ocs - INFO - Adding StorageClass with name storageclass-test-1314213623
14:21:36 - MainThread - utility.utils - INFO - Executing command: oc -n openshift-storage --kubeconfig /home/pbyregow/Music/auth/kubeconfig create -f /tmp/StorageClassd0h4mzrt -o yaml
14:21:37 - MainThread - utility.utils - INFO - Executing command: oc -n openshift-storage --kubeconfig /home/pbyregow/Music/auth/kubeconfig get StorageClass storageclass-test-1314213623 -o yaml
14:21:39 - MainThread - tests.manage.test_rbd_csi_default_sc - INFO - Creating a PVC
14:21:39 - MainThread - resources.ocs - INFO - Adding PersistentVolumeClaim with name pvc-test-1314213929
14:21:39 - MainThread - utility.utils - INFO - Executing command: oc -n openshift-storage --kubeconfig /home/pbyregow/Music/auth/kubeconfig create -f /tmp/PersistentVolumeClaim0iuvw178 -o yaml
14:21:41 - MainThread - utility.utils - INFO - Executing command: oc -n openshift-storage --kubeconfig /home/pbyregow/Music/auth/kubeconfig get PersistentVolumeClaim pvc-test-1314213929 -o yaml
14:21:42 - MainThread - utility.utils - INFO - Executing command: oc -n openshift-storage --kubeconfig /home/pbyregow/Music/auth/kubeconfig get PersistentVolumeClaim pvc-test-1314213929 -o yaml
PASSED                                                                                                                                                                                                      [100%]
------------------------------------------------------------------------------------------------ live log teardown ------------------------------------------------------------------------------------------------
14:21:49 - MainThread - tests.manage.test_rbd_csi_default_sc - INFO - Deleting PVC
14:21:49 - MainThread - utility.utils - INFO - Executing command: oc -n openshift-storage --kubeconfig /home/pbyregow/Music/auth/kubeconfig delete PersistentVolumeClaim pvc-test-1314213929
14:21:51 - MainThread - tests.manage.test_rbd_csi_default_sc - INFO - Deleting StorageClass
14:21:51 - MainThread - utility.utils - INFO - Executing command: oc --kubeconfig /home/pbyregow/Music/auth/kubeconfig delete StorageClass storageclass-test-1314213623
14:21:53 - MainThread - tests.manage.test_rbd_csi_default_sc - INFO - Deleting Secret
14:21:53 - MainThread - utility.utils - INFO - Executing command: oc -n openshift-storage --kubeconfig /home/pbyregow/Music/auth/kubeconfig delete Secret secret-test-1314213323


============================================================================================ 1 passed in 29.01 seconds ============================================================================================

**Pull request merged**